### ISSUE TITLE
Tabs - Accessibility updates

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#701]
**Issue:**
1. No focus indicators.

2. Cannot be utilized via keyboard navigation.

**Solution**:
1. Add focus indicators.

2. Allow component to be used via keyboard navigation.
